libraries/Microsoft.Bot.Builder.Dialogs/DialogManager.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Bot.Builder.Dialogs
     /// <summary>
     /// Class which runs the dialog system.
     /// </summary>
-    public class DialogManager
+    public class DialogManager : IBot
```

Comment:
This is a breaking change.

Reply:
!??

---
In reply to: [603740636](https://github.com/microsoft/botbuilder-dotnet/pull/5389#discussion_r603740636) [](ancestors = 603740636)

Reply:
❓ **REQUIRES JUDGMENT: Adding a new interface implementation to a type**

This is an acceptable change because it does not adversely affect existing clients. Any changes to the type must work within the boundaries of acceptable changes defined here for the new implementation to remain acceptable.